Library/Homebrew/brew.rb

```diff
@@ -6,16 +6,13 @@
   StackProf.start(mode: :wall, raw: true)
 end
 
-raise "HOMEBREW_BREW_FILE was not exported! Please call bin/brew directly!" unless ENV["HOMEBREW_BREW_FILE"]
-
 std_trap = trap("INT") { exit! 130 } # no backtrace thanks
 
 # check ruby version before requiring any modules.
-unless ENV["HOMEBREW_REQUIRED_RUBY_VERSION"]
-  raise "HOMEBREW_REQUIRED_RUBY_VERSION was not exported! Please call bin/brew directly!"
-end
+required_ruby = ENV.fetch("HOMEBREW_REQUIRED_RUBY_VERSION", nil)
+raise "HOMEBREW_REQUIRED_RUBY_VERSION was not exported! Please call bin/brew directly!" unless required_ruby
```

Comment:
Thoughts on just doing:
```suggestion
required_ruby = ENV.fetch("HOMEBREW_REQUIRED_RUBY_VERSION")
```
in these sorts of cases? Not sure it's worth having our own `raise` for the slightly nicer message.